### PR TITLE
change analyst-collective to fishtown-analytics

### PR DESCRIPTION
**Pull request by drewbanin**
s/analyst-collective/fishtown-analytics/g

### CODE REVIEW

File: docs/about/viewpoint.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 > In 2015-2016, a team of folks at RJMetrics had the opportunity to observe, and participate in, a significant evolution of the analytics ecosystem. The seeds of dbt were conceived in this environment, and the viewpoint below was written to reflect what we had learned and how we believed the world should be different. **dbt is our attempt to address the workflow challenges we observed, and as such, this viewpoint is the most foundational statement of the dbt project's goals.**
 
-> The remainder of this document is largely unedited from [the original post](https://medium.com/analyst-collective/building-a-mature-analytics-workflow-the-analyst-collective-viewpoint-7653473ef05b).
+> The remainder of this document is largely unedited from [the original post](https://medium.com/fishtown-analytics/building-a-mature-analytics-workflow-the-fishtown-analytics-viewpoint-7653473ef05b).
```
Comment (jthandy) [inline docs/about/viewpoint.md]: this doesn't resolve

File: dbt/task/init.py [hunk 2]
```diff
@@ -25,8 +25,8 @@
 
 # uncomment below and add real repositories to add dependencies to this project
 #repositories:
-#  - "git@github.com:analyst-collective/repo-name-1"
-#  - "git@github.com:analyst-collective/repo-name-2"
+#  - "git@github.com:fishtown-analytics/repo-name-1"
```
Comment (jthandy) [inline dbt/task/init.py]: can you just make this `[your_org]`

Comment (drewbanin) [inline dbt/task/init.py]: i actually though about that for a second... figured the use case was more `fishtown-analytics/snowplow` than `your-org/your-package`. But sure, np

**Comment by drewbanin** [comment]
@jthandy addressed!

**Pull request merged**